packages/mdc-tooltip/_variables.scss

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/theme/color-palette";
+
+$mdc-tooltip-background: $material-color-grey-700 !default;
+$mdc-tooltip-transition-length-show: 150ms !default;
+$mdc-tooltip-transition-length-hide: 75ms !default;
+$mdc-tooltip-mobil: 750px !default;
```

Comment:
`$mdc-tooltip-mobile`

Where did you get 750px?

Reply:
Just random. Didn't know what the right condition for "Mobile" from the specs is. 
>Also, what is the correct media query for "Mobile" when the tooltips are supposed to have a different size?